<launch>

    <group ns="uav0">
        <node pkg="utm" type="offboard_example" name="offboard_example">
            <remap from="/mavros/state" to="/uav0/mavros/state"/>
            <remap from="/mavros/setpoint_position/local" to="/uav0/mavros/setpoint_position/local"/>
            <remap from="/mavros/setpoint_velocity/cmd_vel" to="/uav0/mavros/setpoint_velocity/cmd_vel"/>
            <remap from="/mavros/cmd/arming" to="/uav0/mavros/cmd/arming"/>
            <remap from="/mavros/set_mode" to="/uav0/mavros/set_mode"/>
        </node>
    </group>

    <group ns="uav1">
        <node pkg="utm" type="offboard_example" name="offboard_example">
            <remap from="/mavros/state" to="/uav1/mavros/state"/>
            <remap from="/mavros/setpoint_position/local" to="/uav1/mavros/setpoint_position/local"/>
            <remap from="/mavros/setpoint_velocity/cmd_vel" to="/uav1/mavros/setpoint_velocity/cmd_vel"/>
            <remap from="/mavros/cmd/arming" to="/uav1/mavros/cmd/arming"/>
            <remap from="/mavros/set_mode" to="/uav1/mavros/set_mode"/>
        </node>
    </group>

</launch>
